.happy/terraform/modules/lambda/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable stack_resource_prefix {
+  type        = string
+  description = "Prefix for account-level resources"
+}
+
+variable image {
+  type        = string
+  description = "Image name"
+}
+
+variable name {
+  type        = string
+  description = "Lambda name"
+}
```

Comment:
i wish there was an easier way to share these variables since a lot of them seem to be repeated

Reply:
We're breaking down the terraform code into smaller modules so unfortunately there isn't a way to avoid repeating the module inputs/outputs :(